templates/vanilla/example/ui/package.json.hbs

```diff
@@ -9,7 +9,7 @@
       "package": "npm run build && cd dist && bestzip ../dist.zip *"
     },
     "dependencies": {
-      "@holochain/client": "^0.12.2",
+      "@holochain/client": "{{holochain_client_version}}",
```

Comment:
Prefer the hardcoded version for now

will create a PR to expose these values to the example templats